src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -29,13 +30,18 @@ public CcdCallbackController(AttachCaseCallbackService attachCaseCallbackService
     }
 
     @PostMapping(path = "/attach_case")
-    public CallbackResponse attachToCase(@RequestBody CallbackRequest callback) {
+
+    public CallbackResponse attachToCase(
+            @RequestBody CallbackRequest callback,
+            @RequestHeader(value = "Authorization") String idamToken,
+            @RequestHeader(value = "user-id") String userId
+    ) {
         if (callback != null && callback.getCaseDetails() != null) {
 
             return attachCaseCallbackService
-                .process(callback.getCaseDetails())
-                .map(modifiedFields -> okResponse(modifiedFields))
-                .getOrElseGet(errors -> errorResponse(errors));
+                    .process(callback.getCaseDetails(), idamToken, userId)
+                    .map(modifiedFields -> okResponse(modifiedFields))
+                    .getOrElseGet(errors -> errorResponse(errors));
```

Comment:
This can be adjusted in:
editor -> code style -> Java -> tabs and indents -> continuation index

(4 instead of 8)